WalletWasabi/WabiSabi/Backend/Rounds/Arena.cs

```diff
@@ -618,16 +618,20 @@ public static ConstructionState AddCoordinationFee(Round round, ConstructionStat
 
 		round.LogInfo($"Available coordination: {availableCoordinationFee}.");
 
-		if (availableCoordinationFee > round.Parameters.AllowedOutputAmounts.Min)
-		{
-			coinjoin = coinjoin.AddOutput(new TxOut(availableCoordinationFee, coordinatorScriptPubKey))
-				.AsPayingForSharedOverhead();
-		}
-		else
+		var minEconomicalOutput = round.Parameters.MiningFeeRate.GetFee(coordinatorScriptPubKey.EstimateOutputVsize()) +
+		                          coordinatorScriptPubKey.EstimateOutputVsize(); // Consider that the output can be spent at 1 s/vb.
```

Comment:
I Think you want InputVsize here and I would suggest to make the fee rate explicit

Reply:
Oh yes good catch I didn't realize